### ISSUE TITLE
Nova is not compatible with password reset changes in laravel 8.57

### ISSUE THREAD

**Issue by mpoma** [laravel/nova-issues#3542]
- Laravel Version: #.#.#
- Nova Version: #.#.#
- PHP Version: #.#.#
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:
https://github.com/laravel/framework/pull/38552
password.reset route not find

**Comment by crynobone** [comment]
As a temporary fix, you may add this on your `NovaServiceProvider`

```php
use Illuminate\Auth\Notifications\ResetPassword;
use Laravel\Nova\Nova;

// ...

Nova::serving(function () {
    ResetPassword::createUrlUsing(function ($notifiable, $token) {
        return route('nova.password.reset', $token);
    });
});
```

**Comment by crynobone** [comment]
This has been fixed via Laravel Framework v8.60.0
